app/models/site_config.rb

```diff
@@ -81,9 +81,10 @@ class SiteConfig < RailsSettings::Base
   field :rate_limit_reaction_creation, type: :integer, default: 10
   field :rate_limit_image_upload, type: :integer, default: 9
   field :rate_limit_email_recipient, type: :integer, default: 5
-  field :rate_limit_article_update, type: :integer, default: 150
+  field :rate_limit_article_update, type: :integer, default: 30
```

Comment:
Just a thought, are we not able to read the default from rate_limit_checker_helper.rb here,  so that they stay in sync?

Reply:
I think we would want to do it the other way around, read THIS default into the helper. I will write that down as a refactor to do in a future PR. Good call!